test/data/3bus/fuel_price.csv

```diff
@@ -1,10 +1,10 @@
 genfuel,area,subarea,price,quantity
 coal,,,0.1,Inf
-ng,nation,narnia,1.5,5000
-ng,nation,narnia,1.75,5000
-ng,nation,narnia,2.0,5000
-ng,nation,narnia,2.25,Inf
-ng,nation,archenland,0.1,50000
-ng,nation,archenland,0.2,50000
-ng,nation,archenland,0.3,50000
-ng,nation,archenland,0.4,Inf
\ No newline at end of file
+ng,bus_nation,narnia,1.5,5000
```

Comment:
same as with af, isn't it better to use just nation here instead of bus_nation? this isn't consequential for these tests but I'm thinking about our standard inputs and what would be appropriate for them

Reply:
See comment above.